tobac/tests/test_segmentation.py

```diff
@@ -1,5 +1,7 @@
 import tobac.testing as testing
 import tobac.segmentation as seg
+import tobac.feature_detection as feat_detect
```

Comment:
Could we change these to import the full names for clarity? e.g. `from tobac import segmentation, feature_detection`

Reply:
Agreed, I changed this